client/src/components/squidIndex/SquidTile.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+export const SquidTile = ({ squid }) => {
+  const specialPowerElement = squid.specialPower ? <p>Special Power: {squid.specialPower}</p> : null
+  const hrLineElement = squid.id === "1" ? "" : <hr />
```

Comment:
hm - we can't rely on a hard `id` value in production. If we want to separate the tiles I would suggest we do so in the `List` component. 

```javascript
squidListQueryOutput = squids.map((squid) => <div key={squid.id}><SquidTile squid={squid} /><hr /></div>)
```